package/Dockerfile

```diff
@@ -55,7 +55,7 @@ ENV CATTLE_SYSTEM_AGENT_UPGRADE_IMAGE rancher/system-agent:${CATTLE_SYSTEM_AGENT
 ENV CATTLE_SYSTEM_UPGRADE_CONTROLLER_CHART_VERSION 103.0.0+up0.6.0
 
 # System charts minimal version
-ENV CATTLE_FLEET_MIN_VERSION=103.1.0+up0.9.0-rc.3
+ENV CATTLE_FLEET_VERSION=103.1.0+up0.9.0-rc.3
```

Comment:
Word of warning that we want to be cautious about renaming settings. I am not sure how/if customers use this setting today, but if it is something heavily used we may need to deprecate the setting for a release before it is removed.

Reply:
Indeed, as discussed we're keeping `CATTLE_FLEET_MIN_VERSION` which, if set, will have precedence over `CATTLE_FLEET_VERSION`. In any case, install behaviour will be the same, ie Fleet will be installed with a _min version_ (whichever setting it comes from), looking for an exact match to prevent the latest version from being installed instead, but keeping any newer version if already installed.